e2e/Makefile

```diff
@@ -0,0 +1,8 @@
+
+.PHONY: test
+test: tools
+	go test ./...
```

Comment:
Can we use here `gotestsum`, as we already do in https://github.com/testcontainers/testcontainers-go/pull/451/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR55? 🙏 

This way we will be using the same tools and generating the test reports in the same formats

Reply:
done